src/main/resources/Dockerfile

```diff
@@ -1,9 +1,5 @@
-FROM centos:7 AS tarantool-base
+FROM centos:7
 ARG TARANTOOL_VERSION=2.8
-ARG TARANTOOL_SERVER_USER="tarantool"
```

Comment:
This is a w/a for Ubuntu and some other systems that don't allow running Tarantool as root. I wouldn't remove this without an evidence that the new approach works everywhere.

And it is worth it, please fix the README too.

Reply:
Ok. I reverted the params back, but changed to root by default.
The combination of running tarantool as root + copying files is working as expected on all systems - it should be the default one.